cg/server/api.py

```diff
@@ -25,12 +27,49 @@
 from google.auth import jwt
 from pydantic import ValidationError
 from requests.exceptions import HTTPError
-from sqlalchemy.orm import Query
 from werkzeug.utils import secure_filename
 
 LOG = logging.getLogger(__name__)
 BLUEPRINT = Blueprint("api", __name__, url_prefix="/api/v1")
 
+cache = TTLCache(maxsize=1, ttl=3600)
+
+
+def get_certificate_ttl(response_data):
+    expires_header = response_data.headers.get("Expires")
+    expires = datetime.strptime(expires_header, "%a, %d %b %Y %H:%M:%S %Z")
+    ttl = int((expires - datetime.utcnow().replace(tzinfo=timezone.utc)).total_seconds())
+
+    return ttl
```

Comment:
Docstring and type hints
```suggestion
def get_certificate_ttl(response_data) -> int:
    """Returns the time-to-live of given response."""
    expires_header: str = response_data.headers.get("Expires")
    expires: datetime = datetime.strptime(expires_header, "%a, %d %b %Y %H:%M:%S %Z")
    ttl: int = int((expires - datetime.utcnow().replace(tzinfo=timezone.utc)).total_seconds())
    
    return ttl
```